### PR TITLE
[MIG] l10n_ch_qr_bill_scan: Migration to 14.0

### PR DESCRIPTION
**Pull request by matiasperalta1**

### CODE REVIEW

**Comment by TeoGoddet** [comment]
Hello @matiasperalta1, what is the status of this PR ?

File: l10n_ch_qr_bill_scan/wizard/account_invoice_import.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+try:
+    import cv2
+    import numpy as np
+    import pdf2image
+    from pyzbar.pyzbar import ZBarSymbol, decode
+except ImportError:
+    # Necessary libraries to decode QR from pdf are
+    # not installed
+    logger.warning(
```
Comment (TeoGoddet) [inline l10n_ch_qr_bill_scan/wizard/account_invoice_import.py:22]: Maybe we should also log the original error to help troobleshooting (in my case some missing system packages such as python3-opencv libzbar0 poppler-utils

File: l10n_ch_qr_bill_scan/readme/INSTALL.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+This module depends on `account_invoice_import` which is located in `edi` and has dependencies on `community-data-files`.
+So the repository `edi` and `community-data-files` must be present on the system.
+
+Optionally you can also import Vendor bills directly from a PDF.
+To enable this feature those additional dependencies must be
+installed on your Odoo server:
+
+    pip install pyzbar,numpy,pdf2image,opencv-python
+
```
Comment (TeoGoddet) [inline l10n_ch_qr_bill_scan/readme/INSTALL.rst:9]:
```suggestion
Some system library are also required (opencv, libzbar and poppler) :
On debian, run apt install python3-opencv libzbar0 poppler-utils 
```

**Comment by TeoGoddet** [comment]
Tested it, it work well,

**Comment by TeoGoddet** [comment]
Should the reference field go to payment_reference or ref (or both ?)

**Comment by i-vyshnevska** [comment]
travis is broken, also seems some parts are missing
for instance, if we provide invoice scan with qr file is not required anymore, but it still mandatory in the wizard
this part is missing
```

<field name="invoice_file" position="attributes" >
    <attribute name="required">0</attribute>
</field>
```

**Comment by TeoGoddet** [comment]
My guess is travis should be replaced by github actions and runbot by runboat
This need an update of the dotfiles in this repo

**Comment by TeoGoddet** [comment]
I checked, the dotfile seems ok, 
this pr should be rebased
I can't do as i'm not maintainer
@matiasperalta1 do you want to handle this or maybe we should open another pr

File: l10n_ch_qr_bill_scan/wizard/account_invoice_import_view.xml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        <field
+            name="inherit_id"
+            ref="account_invoice_import.account_invoice_import_form"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='help-import']" position="before">
+                <div name="help-import-scan" states="import">
+                    <p><big>
+                            <big>
+                                <i class="fa fa-qrcode" />
```
Comment (i-vyshnevska) [inline l10n_ch_qr_bill_scan/wizard/account_invoice_import_view.xml:39]:
```suggestion
                                <i class="fa fa-qrcode" title="QR-scan"/>
```

**Comment by i-vyshnevska** [comment]
@matiasperalta1 pls rebase to trigger build

**Comment by ajaniszewska-dev** [comment]
@matiasperalta1 ping, can you rebase this one so we can merge? That would help a bit with migration to v15.

**Comment by ajaniszewska-dev** [comment]
note:
looks need dependency module migrated (account_invoice_import):
https://github.com/OCA/edi/pull/496 or https://github.com/OCA/edi/pull/559

**Pull request closed without merging**